### PR TITLE
Issue 739 by pjohans: event validate handler to check if selected curren...

### PR DESCRIPTION
**Pull request by pjohans**
...cy is supported by place2book

as suggested by kasperg

### CODE REVIEW

**Comment by kasperg** [comment]
@pjohans: The file `includes/ding_place2book.event_validate.inc` seems to be missing from the pull request. Did you forget to add it?

**Comment by pjohans** [comment]
yes i did

File: includes/ding_place2book.event_validate.inc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+function ding_place2book_form_ding_event_admin_currency_setting_validate($form, &$form_state){
+  $currency = $form_state['values']['ding_event_currency_type'];
+  if (module_exists('ding_place2book') && $currency != 'kr') {
+    drupal_set_message(t('place2book does not support currencies other than Kr. disable ding_place2book module to continue'), 'warning');
+    $form_state['input']['ding_event_currency_type'] = 'kr';
+    $form_state['rebuild'] = TRUE;
+    return;
+  }
+}
```
Comment (kasperg) [inline includes/ding_place2book.event_validate.inc]: There is a missing newline at the end of the file.

**Comment by kasperg** [comment]
Reviewed and approved.

**Pull request merged**